_spinnaker_install_admin_guides/install-on-aws-halyard-pod.md

```diff
@@ -0,0 +1,668 @@
+```
+
+## Set up an Ingress for `spin-deck` and `spin-gate`
+
+Get the external IP for the NGINX ingress controller:
+
+```
+kubectl get svc -n ingress-nginx
+```
+
+You'll get a DNS name or IP address in the `EXTERNAL-IP` field.
+
+If you stood up a new NGINX ingress controller, you can likely just this value (IP address or DNS name) for your ingress.
+
+For example, if I get `abcd1234abcd1234abcd1234abcd1234-123456789.us-west-2.elb.amazonaws.com`, then I can do the following:
+* Use `abcd1234abcd1234abcd1234abcd1234-123456789.us-west-2.elb.amazonaws.com` for my SPINNAKER_ENDPOINT in the below steps
+
+For example, if I get `55.55.55.55`, then I can do the following:
+* Use ``55.55.55.55` for my SPINNAKER_ENDPOINT in the below steps
+
+If you are using an existing NGINX ingress controller, or other services are likely to be using the same NGINX ingress controller, you should create a DNS entry that points at your NGINX ingress controller endpoint. (either a `CNAME Record` that points at the DNS name, or an `A Record` that points at the IP address).
+
+For example, if I get `abcd1234abcd1234abcd1234abcd1234-123456789.us-west-2.elb.amazonaws.com`, then I can do the following:
+* Create a CNAME pointing `spinnaker.domain.com` at `abcd1234abcd1234abcd1234abcd1234-123456789.us-west-2.elb.amazonaws.com`
+* Put `spinnaker.domain.com` in the `host` field in the below manifest (and uncomment it)
+* Use `spinnaker.domain.com` for my SPINNAKER_ENDPOINT in the below steps
+* (Alternately, for testing, create an `/etc/hosts` entry pointing spinnaker.domain.com at the IP address that `abcd1234abcd1234abcd1234abcd1234-123456789.us-west-2.elb.amazonaws.com` resolves to)
+
+If I get `55.55.55.55`, then I can do the following:
+* Create n A Record pointing `spinnaker.domain.com` at `55.55.55.55`
+* Put `spinnaker.domain.com` in the `host` field in the below manifest (and uncomment it)
+* Use `spinnaker.domain.com` for my SPINNAKER_ENDPOINT in the below steps
+* (Alternately, for testing, create an `/etc/hosts` entry pointing `spinnaker.domain.com` at `55.55.55.55`)
+
+Create a Kubernetes Ingress manifest to expose spin-deck and spin-gate (change your hosts and namespace accordingly):
+
+```bash
+tee spin-ingress.yaml <<-'EOF'
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: spin-ingress
+  labels:
+    app: spin
+    cluster: spin-ingress
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+spec:
+  rules:
+  - 
+    # host: spinnaker.some-url.com
+    # ^ If you have other things running in your cluster, you should uncomment this line and specify a valid DNS name
+    http:
+      paths:
+      - backend:
+          serviceName: spin-deck
+          servicePort: 9000
+        path: /
+      - backend:
+          serviceName: spin-gate
+          servicePort: 8084
+        path: /api/v1
+EOF
+```
+
+Create the Ingress
```

Comment:
Nitpicky - but maybe "Apply" is more appropriate.

Reply:
fixed